tests/test_probability.py

```diff
@@ -1,9 +1,10 @@
+import pyhf
 from pyhf import probability
 import numpy as np
 
 
 def test_poisson(backend):
-    tb, _ = backend
+    tb, _ = pyhf.get_backend()
```

Comment:
this should not have been changed. We have to inject the backend this way for testing purposes (if the tests are in weird orders, the `pyhf.get_backend()` isn't properly reset for tests).